Gemfile

```diff
@@ -10,30 +10,30 @@ end
 gem 'rails', '~> 5.1.2'
 
 gem 'jbuilder', '~> 2.5'
+gem 'kaminari'
```

Comment:
WTF why can't ruby people use reasonable names for anything

Reply:
It practically screams " I AM A PAGINATION LIB WITH ACTIVE RECORD EXTENSIONS" so I'm not sure I understand what you mean